### ISSUE TITLE
[BR] New releases remove previous docker images and tags

### ISSUE THREAD

**Issue by InputObject2** [kubealex/k8s-mediaserver-operator#38]
**Describe the bug**
Every time a new version of the image is pushed, the previous versions disappear from quay.io, leading to errors such as:

```
Failed to pull image "quay.io/kubealex/k8s-mediaserver-operator:v0.6": rpc error: code = Unknown desc = Error response from daemon: unknown: Tag v0.6 was deleted or has expired. To pull, revive via time machine
```

The history on quay.io for this images indicates that the previous tags are indeed being deleted: https://quay.io/repository/kubealex/k8s-mediaserver-operator?tab=history

**Expected behavior**
There are two usual behaviors that are expected when using semver and docker images:
- Previous tags should be immutable and not purged on new versions.
- Multiple tags should be attached to each release. For exemple for `0.6.1` (current latest), we should expect tags `latest`, `0`, `0.6` and `0.6.1`. Users would then be able to specify a tagged version and expect it to continue working throughout new releases.

**Additional context**
This seems like something that should be modified in the `docker-build` process in the Makefile but I haven't figured out why it is purging previous images.

**Comment by kubealex** [comment]
Due to some incompatibilities of some modifications with the new manifests, I had to manually remove the previous tags to prevent people using bugged versions if not using the "Always" pull-policy. 
I am slightly changing your suggestion to fit the repo needs :)

**Comment by InputObject2** [comment]
Solved by #41
